src/query/src/promql/planner.rs

```diff
@@ -1035,8 +1035,17 @@ impl PromPlanner {
             .context(DataFusionPlanningSnafu)?;
 
         // make divide plan
+        let time_index_column = self
+            .ctx
+            .time_index_column
+            .clone()
+            .with_context(|| TimeIndexNotFoundSnafu { table: "unknown" })?;
```

Comment:
Can we use the table_ref?
```suggestion
            .with_context(|| TimeIndexNotFoundSnafu { table: table_ref.to_string() })?;
```